webapp/channels/src/actions/invite_actions.ts

```diff
@@ -156,18 +157,36 @@ export async function sendGuestInviteForUser(
     teamId: string,
     channels: Channel[],
     members: RelationOneToOne<Channel, Record<string, ChannelMembership>>,
+    getState: () => GlobalState,
 ): Promise<({sent: InviteResult} | {notSent: InviteResult})> {
+    const state = getState();
+    const currentUserIsAdmin = isCurrentUserSystemAdmin(state);
```

Comment:
instead of passing the whole state to the function, we can use the `isSystemAdmin` from `user_utils` to check it user is admin or not.

Reply:
Roger